pkg/cfn/builder/partition.go

```diff
@@ -26,24 +26,25 @@ var servicePrincipalPartitionMappings = map[string]map[string]string{
 
 const servicePrincipalPartitionMapName = "ServicePrincipalPartitionMap"
 
-func makeFnFindInMap(mapName string, args ...*gfn.Value) *gfn.Value {
-	return gfn.MakeIntrinsic(gfn.FnFindInMap, append([]*gfn.Value{gfn.NewString(mapName)}, args...))
+// TODO put this in gfn
```

Comment:
Reminer: should this happen before or after this PR?

Reply:
Oh nice catch, fixed it now.